package METRO.CRO.MESSAGES;

message ClassCategory {
	required string u_id = 1;
	required string name = 2;
	optional string description = 3;
	optional string imgurl = 4;
}